integration-tests/actions/ocr_helpers.go

```diff
@@ -415,3 +422,128 @@ func BuildNodeContractPairID(node *client.Chainlink, ocrInstance contracts.Offch
 	shortOCRAddr := ocrInstance.Address()[2:12]
 	return strings.ToLower(fmt.Sprintf("node_%s_contract_%s", shortNodeAddr, shortOCRAddr)), nil
 }
+
+func BuildGeneralOCR2Config(
```

Comment:
so this could not be applied to VRF?

Reply:
@iljapavlovs it probably can be applied to vrf. Can we do it in a separate ticket?

Reply:
sure, just for me OCR config was different compared to Automation